harvdev_utils/psycopg_functions/sql_queries.py

```diff
@@ -0,0 +1,329 @@
+"""
+.. module:: sql_queries
+   :synopsis: Commonly used sql queries. Use ".format()" function to add values for "{}" variables.
+
+.. moduleauthor:: Gil dos Santos dossantos@morgan.harvard.edu
+"""
+
+# Get all current, non-analysis features given a uniquename regex.
+current_features = """
+    SELECT DISTINCT f.feature_id,
```

Comment:
good use of  """ """. Makes reading the sql much easier :-)
Though i had try to be positive in my review too.

Reply:
aw shucks